Build.fsx

```diff
@@ -69,7 +69,7 @@ Target "PatchAssemblyVersions" (fun _ ->
         buildVersion.fileVersion
         buildVersion.infoVersion
 
-    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.*"
+    let filesToPatch = !! "Src/*/Properties/AssemblyInfo.fs"
```

Comment:
Well, we use commit messages for such information. You can always use `git blame` to find out information about particular line. In this particular case I've duplicated the commit message in the code. While I agree that it might look fine in this case, I don't feel like it might sense to do that always.